webrender_traits/src/webgl.rs

```diff
@@ -307,6 +314,8 @@ impl WebGLCommand {
                 gl::uniform_matrix_4fv(uniform_id, transpose, &v),
             WebGLCommand::UseProgram(program_id) =>
                 gl::use_program(program_id.get()),
+            WebGLCommand::ValidateProgram(program_id) =>
+                gl::validate_program(program_id.get()),
```

Comment:
No. Yo have to check the result using a separate function. I already tried in a JavaScript demo:

gl.getProgramParameter( program, gl.VALIDATE_STATUS)